docker-compose.yaml

```diff
@@ -9,12 +9,39 @@ services:
       dockerfile: Dockerfile
     ports:
       - "8080:3000"
+    depends_on:
+      - robin-db
+    environment:
+      DB_HOST: db
+      DB_PORT: 3306
+      DB_USER: root
+      DB_PASSWORD: password
+      DB_NAME: robin-development
     volumes:
       - ./configs:/root/configs
     restart: unless-stopped
     networks:
       - rha_local
 
+  robin-db:
+    container_name: robin-db
+    image: mysql:8.4.0
+    cap_add:
+      - SYS_NICE
+    restart: always
+    environment: 
+      - MYSQL_DATABASE=robin-development
+      - MYSQL_ROOT_PASSWORD=password
+    volumes:
+    - db-data:/var/lib/mysql
+    - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
`init.sql` file missing in the PR

Reply:
Not sure, if we need this.

Reply:
suggested because file was referenced here but was missing from PR. Can remove if it's not needed